rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/util/FileExtensions.kt

```diff
@@ -52,4 +52,5 @@ fun isGeneratedUnityFile(file: VirtualFile): Boolean {
     return extension.equals("csproj", true) || extension.equals("sln", true)
 }
 
-fun isUxmlFile(file: VirtualFile) = file.extension.equals("uxml", true)
\ No newline at end of file
+fun isUxmlFile(file: VirtualFile) = file.extension.equals("uxml", true)
```

Comment:
it's a better practice to operate with file types rather than with actual hard-coded strings comparison. custom file associations can be added, there could be another extension in the future etc. 

This is not a big problem for your scenario, and you also covered the 'ignore case' thing, but still...

Consider using `FileTypeRegistry#isFileOfType` is possible